### DISCUSSION TITLE
Bug: File Finder input field focus on Firefox

### DISCUSSION THREAD

**Discussion by amercader** [community/community#22209] [New to GitHub]
<p>Hi GitHub support team, I’d like to report a bug in the File Finder feature.</p>
<p>When browsing the site on Firefox while logged in and typing the letter <code>t</code> to look for a file, the file list appears but the cursor is not focused on the input box to write the file name.</p>
<p><img src="https://user-images.githubusercontent.com/200230/181069522-85f47502-611f-45fe-bab8-0c082acedffb.png" alt="Screenshot_2020-11-25 ckan ckan" data-base62-sha1="5RJYb223I75IN5TmzPPPVELSh9X" width="308" height="202"></p>
<p>Repeating the same process when not logged in shows the expected behaviour (ie input box gets focused after typing <code>t</code>)<br>
Chrome shows the expected behaviour both when logged in and when not logged in.</p>
<p>So to sum up:</p>
<ul>
<li>Firefox + Logged in = Bug (input not focused)</li>
<li>Firefox + Not Logged in = Works (input focused)</li>
<li>Chrome + Not Logged in = Works (input focused)</li>
<li>Chrome + Not Logged in = Works (input focused)</li>
</ul>
<p>Many thanks</p>

**Comment by mickaelpham** [comment]
<p>I would like to upvote this issue. Was able to reproduce it with the same summary as the original poster:</p>
<ul>
<li>Firefox + logged in = bug</li>
<li>Firefox + not logged in = works</li>
</ul>

**Comment by chris-smith-zocdoc** [comment]
<p>I’m seeing the same behavior. Firefox + logged in results in the loss of focus on the file name input box</p>

**Comment by mickaelpham** [comment, marked as answer]
<p>As if 2020-12-03, I’m no longer seeing the error anymore.</p>
<p>Logged in, Firefox user, press T -&gt; focus is good</p>

**Comment by amercader** [comment]
<p>I can confirm it works for me too. Marking as fixed, thanks!</p>
<p>edit: I can’t edit the title to add [FIXED]</p>
